### PR TITLE
Upload Image on s3

### PR DESCRIPTION
**Pull request by achyutkadamacl**

### CODE REVIEW

File: app/controllers/welcomes_controller.rb [hunk 1]
```diff
@@ -1,4 +1,22 @@
 class WelcomesController < ApplicationController
   def index
   end
+
+  def download
+    puts"||||||||||||||||||||||||"
```
Comment (theachyutkadam) [inline app/controllers/welcomes_controller.rb:6]: Remove unwanted puts message.

File: app/views/welcomes/index.html.erb [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+<br><br><br>
+
+<h1>Copy bucket</h1>
+<%= form_with scope: "welcomes", url: send_data_into_another_bucket_path, method: :get do |form| %>
+	<%= form.label :first_bucket, "Select First Bucket:" %>
+  <%= form.text_field :first_bucket %><br><br>
+	<%= form.label :second_bucket, "Select Second Bucket:" %>
+  <%= form.text_field :second_bucket %><br><br>
+  <%= form.submit "Paste" %>
+<% end %>
```
Comment (theachyutkadam) [inline app/views/welcomes/index.html.erb:20]: Add a end line.

**Review by theachyutkadam — APPROVED** [review]
Remove test folder, and also setup a rspec for testing.

File: app/views/welcomes/index.html.erb (hunk 2 above)
Comment (achyutkadamacl) [reply app/views/welcomes/index.html.erb:20]: Sure @theachyutkadam  I'll update it, next branch.

**Pull request merged**